python/sglang/utils.py

```diff
@@ -540,3 +543,114 @@ def resolve_obj_by_qualname(qualname: str) -> Any:
     module_name, obj_name = qualname.rsplit(".", 1)
     module = importlib.import_module(module_name)
     return getattr(module, obj_name)
+
+
+class CachedKernel:
```

Comment:
Can we implement this in https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/utils.py instead

Reply:
yes, please move  it to srt/utils.py
the `sglang/utils.py` is mostly for the language frontend